videos/videos.js

```diff
@@ -31,94 +31,111 @@ router.get("/:id", async (req, res) => {
 // videos: video_title, location
 // We'll want to adjust the model to allow for both
 
-
 router.post("/", restricted, async (req, res) => {
   // this should be unnecessary because we're rebuilding objects
   // const data = req.body;
-  const { title_short, preview, artist, deezer_id, location, video_title, user_id } = req.body;
+  const {
+    title_short,
+    preview,
+    artist,
+    deezer_id,
+    location,
+    video_title,
+    user_id,
+  } = req.body;
 
   const songObject = {
     deezer_id: deezer_id,
     title: title_short,
-    artist_name: artist
-  }
+    artist_name: artist,
+  };
 
   const videoObject = {
     video_title: video_title,
     location: location,
     song_id: "",
-    user_id: user_id
+    user_id: user_id,
   };
 
   // console.log(videoObject);
+  // console.log(songObject);
 
   try {
-        // Start with request to DS server, should get a 200 immediately if working
-        // Then go to rest of functions
-        // Potentially explore 
-
-        const song = await Songs.add(songObject);
-        // console.log(song);
-
-        const videoObjectComplete = { ...videoObject, song_id: song };
-
-        // console.log(videoObjectComplete);
-
-
-
-        // console.log(videoObjectComplete);
-        // we want song to return its id
-        // then we'll destructure or spread some object to add that in
-        // then we'll 'add' that object to videos.add
-        const video = await Videos.add(videoObjectComplete);
-
-        axios
-          .post(
-            "http://sample.eba-5jeurmbw.us-east-1.elasticbeanstalk.com/entry", null,
-            {
-              params: {
-                preview: preview,
-                video_id: video,
+    if (!title_short) {
+      res.status(400).json({ message: "Missing title_short!" });
```

Comment:
Damn look at all those if/elses! Is there any way for us to do this that isn't such a deep tree of if/else statements? Seems like there should be...

Reply:
I am sure that there is a way to do it.  I'm just not sure what it is.  I do know that this is working though.  I am not proud though and definitely not married to any of the code that I've laid down. If anyone has suggestions on how to get this done is a cleaner way I would love to get my code cleaned up.